packages/serp-report/src/pages/iframe/wtm-report.js

```diff
@@ -36,121 +36,159 @@ function requestClose() {
   window.parent.postMessage('WTMReportClosePopups', '*');
 }
 
+function disable() {
+  window.parent.postMessage('WTMReportDisable', '*');
+}
+
 export default define({
   tag: 'wtm-report',
-  render: () => html`
-    ${html.resolve(
-      Stats.then(
-        (stats) => html`
-          <panel-header domain=${stats.domain}>
-            <button class="svg-button" onclick="${requestClose}">
-              ${close}
-            </button>
-          </panel-header>
-        `,
-      ),
-    )}
+  confirmDisabled: false,
+  render: ({ confirmDisabled }) => html`
+    <panel-header domain=${domain}>
+      <button class="svg-button" onclick="${requestClose}">${close}</button>
+    </panel-header>
 
     <main>
-      <h1>${t('android_site_blocking_header')}</h1>
+      <h1>${t('wtm_trackers_preview_title')}</h1>
```

Comment:
That would need to be added as a translation at a difference place, right (ghostery-extension + ghostery-dnr-extension)?

Reply:
yes, we sync translation files with transifex

Reply:
Unfortunately, we don't have yet process to keep translations here, so they must be provided by the applications. And we have a PR to DNR, that have them.

Reply:
actually, for simplicity we could have all transifex translation in this repo as well.